tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -66,8 +66,13 @@ function configure_cri_runtime() {
 		configure_containerd
 		;;
 	esac
-	systemctl daemon-reload
-	systemctl restart "$1"
+	
+	if [ -x /sbin/openrc-run ]; then
+		service k3s-service restart
```

Comment:
We don't want to restart k3s-service, we want to restart "$1", which may be crio, containerd, k3s, k3s-agent.  See the suggestion below, please.

Reply:
I check k3s install script, it named the service as k3s-service, might be that need to taken into account?

Reply:
It might be taken into account, and you should also check the name on different distros supporting OpenRC.

Reply:
Sorry I don't have resource for doing so.  Providing many distribution running systemd, OpenRC only used in some lightweight systems, I think install.sh is likely the main way to get k3s install, hence get cover with this change.